content/news/2023-08-09-radar-becomes-a-maplibre-silver-sponsor/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Radar becomes a MapLibre Silver Sponsor"
+date: "2023-08-09"
+categories: ["announcements"]
+authors: [duje]
+draft: false
+---
+
+Radar is joining the MapLibre Sponsorship Program as a Silver Sponsor.
+
+<img src="https://user-images.githubusercontent.com/23352538/259137907-61713b8a-2221-4d2c-9dc4-d5ca916e6949.svg" width="100%"/>
```

Comment:
Please make sure this image is in format 4:3 or so, otherwise Twitter will crop it.

Reply:
Is the background white now?

Reply:
I would advise you to use PNG, then you know what you get.

Reply:
a bit of padding would be nice

Reply:
your call, but imo it still needs some padding - and it would be nice to align the text

<img width="894" alt="Screenshot 2023-08-11 at 18 06 09" src="https://github.com/maplibre/maplibre.github.io/assets/74932975/1d82f772-1252-4d34-9810-ed9f888d2971">

Reply:
Sure, here is one.

<img width="1381" alt="Screenshot 2023-08-11 at 18 38 16" src="https://github.com/maplibre/maplibre.github.io/assets/74932975/4b267b2c-a4f9-4be1-8997-2b8c1d88825e">

Found it's somewhat cleaner to have the "Radar is joining the MapLibre Sponsorship Program as a Silver Sponsor" below the image with the other text so it's not dangling above:

<img width="686" alt="Screenshot 2023-08-11 at 18 40 59" src="https://github.com/maplibre/maplibre.github.io/assets/74932975/7e1ce770-c483-4260-b84f-7351af92f934">